README.md

```
@@ -178,6 +175,10 @@ For repeatable Docker deploys, prefer the versioned image tag from the release p
 ![](.github/screenshots/screenshot_8.png)
 ![](.github/screenshots/screenshot_9.png)
 ![](.github/screenshots/screenshot_10.png)
+![](.github/screenshots/screenshot_11.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)